package/src/module.js

```diff
@@ -160,13 +169,10 @@ function isList(el) {
 }
 
 // Future functionality: Ordered Lists
-function isOrderedList(el) {
+function getListType(el) {
   const val = el.textContent[0]
   const regex = /^\d+$/
-  if (regex.test(val)) {
-    return true
-  }
-  return false
+  return regex.test(val) ? 'OL' : 'UL'
```

Comment:
changed the name to represent that it will return the list type, not just a boolean.